### ISSUE TITLE
circular-json in maintenance

### ISSUE THREAD

**Issue by hozefaj** [royriojas/flat-cache#22]
Getting the below warning when `circular-json` is installed as a dependency.

```
CircularJSON is in maintenance only, flatted is its successor.
```

Maybe worth while to update dependency to use `flatted`....https://github.com/WebReflection/flatted

**Comment by ai** [comment]
Fix https://github.com/royriojas/flat-cache/pull/23

**Comment by royriojas** [comment]
closed by #23

**Comment by royriojas** [comment]
Sadly this broke the eslint tests (not sure how) but it is most likely an issue with the way the cache is generated. We're missing a unit test there.

**Comment by hudochenkov** [comment]
It also broke stylelint tests: https://github.com/stylelint/stylelint/issues/3779#issuecomment-436687938 (more info about flat-cache update there). It's kind of our fault, we test implementation detail of dependency of a dependency of a dependency. In our case, it's output of `circular-json`, which is a dependency of `flat-cache`, which is a dependency of `file-entry-cache`.

@royriojas there is no your fault. `file-cache` works with `flatted` as expected. Both `stylelint` and `eslint` worked with `flat-cache@1.3.1` good. It's just our tests doing bad things :)

**Comment by cktang88** [comment]
closed by #35

**Comment by AlexWayfer** [comment]
> closed by #35

By duplicated issue?

I think, it closed by #36.
